src/test/resources/hudson/tasks/configuration-as-code.yaml

```diff
@@ -0,0 +1,15 @@
+tool:
+  ant:
+    defaultProperties:
```

Comment:
this shouldn't be required, it's a UI only property that jcasc should be ignoring (it's blacklisted in jcasc, if this isn't working possibly the blacklist needs an update)

Reply:
But I saw https://issues.jenkins-ci.org/browse/JENKINS-57122, which is related, and this way that issue is tested as well.

Reply:
I wouldn't put bad properties in the main test, 
the test should cover that it imports the expected config and the config that is exported is what's expected.

That also covers the default properties as you're asserting that expected config

Reply:
You are more experienced on JCasC than me, so if you are sure that it is covered, I will 100% trust on you. I will remove it!

Reply:
default properties should not be exported indeed.